src/Cache/ProductionCache.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+namespace Torr\Rad\Cache;
+
+use Symfony\Contracts\Cache\CacheInterface;
+
+/**
+ * A cache wrapper, that disables the cache in debug once per request
+ */
+final class ProductionCache implements CacheInterface
+{
+	/** @var array<string, bool> */
+	private array $alreadyRequested = [];
+
+	/**
+	 */
+	public function __construct (
+		private readonly CacheInterface $cache,
+		private readonly bool $isDebug,
```

Comment:
Should we try to integrate this into the `HostingEnvironment`?

Reply:
No, it is directly integrated into `APP_DEBUG`. If we are in debug, we want to allow a miss per request, otherwise not.

Our stage should use the same logic as prod = always properly cache, as all other things are cached as well.

Reply:
Maybe that makes more sense for one our bridges, if we don't want to require `21torr/hosting` directly.

Reply:
The purpose of this class is actually something else.. I will rename it, to make it clearer.

The persistence using the cache library is actually not necessary.